control-plane/config/rbac/role.yaml

```diff
@@ -29,285 +29,19 @@ rules:
-  resources:
-  - jwtproviders/status
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
   - mesh
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
-  - mesh/status
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
   - peeringacceptors
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
-  - peeringacceptors/status
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
   - peeringdialers
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
-  - peeringdialers/status
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
   - proxydefaults
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
-  - proxydefaults/status
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
   - samenessgroups
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
-  - samenessgroups/status
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
   - servicedefaults
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
-  - servicedefaults/status
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
   - serviceintentions
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
-  - serviceintentions/status
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
   - serviceresolvers
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
-  - serviceresolvers/status
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
   - servicerouters
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
-  - servicerouters/status
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
   - servicesplitters
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
-  - servicesplitters/status
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - consul.hashicorp.com
-  resources:
   - terminatinggateways
```

Comment:
This formatting is just consolidated now, you don't have to add the verbs for each resource type